tests/dmctl_basic/run.sh

```diff
@@ -131,6 +131,9 @@ function run() {
 
     dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
     dmctl_operate_source create $WORK_DIR/source2.yaml $SOURCE_ID2
+    run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+        "operate-source show" \
```

Comment:
How about showing detailed results both for _all sources bouned_ and _some sources not bouned_?

Reply:
https://github.com/pingcap/dm/pull/838/commits/2ba6fa215eaeeaa3897c33b8dca0c8b33b5e4308 added test